packages/containers/src/Notification/Notification.actions.js

```diff
@@ -0,0 +1,49 @@
+import Constants from './Notification.constant';
+
+/**
+ * Action to push a generic notification
+ * @param {Object} notification Notification to push
+ * @returns {Object}
+ */
+export function pushGeneric(notification) {
+	return {
+		type: Constants.PUSH_NOTIFICATION,
+		notification,
+	};
+}
+
+/**
+ * Action to specifically push an info notification
+ * @param {Object} notification Notification to push
+ * @returns {Object}
+ */
+export function pushInfo(notification) {
+	return pushGeneric({
+		...notification,
+		type: 'info',
```

Comment:
I think that it could be nice to export `info`, `warning`, etc. as constants in the component. WDYT ?